ddtrace/internal/coverage/lines.py

```diff
@@ -18,28 +28,30 @@ def __repr__(self):
         return f"CoverageLines(num_lines={self._num_lines()})"
 
     def _num_lines(self) -> int:
-        return sum([bin(byte).count("1") for byte in self._lines])
+        return sum(_bit_count(byte) for byte in self._lines)
 
     def add(self, line_number: int):
         lines_byte = line_number // 8
-        lines_bit = 2 ** (line_number % 8)
 
         if lines_byte >= len(self._lines):
             self._lines.extend(bytearray(lines_byte - len(self._lines) + 1))
 
+        # DEV this fun bit allows us to trick ourselves into little-endianness, which is what the backend wants to see
+        # in bytes
+        lines_bit = 2 ** (7 - (line_number % 8))
```

Comment:
I still think it would be clearer to do `0b10000000 >> (line_number % 8)` (you can think about it visually without even thinking about bits, as moving the `1` to where you want it to be). But this is Fine™.